docs: project: dc2677a: Add doc

actorreno opened the pull request:
Adding documentation for DC2677A. Part of migration

## PR Description

This PR adds the docs for the project DC2677A.


## PR Type
- [ ] Bug fix (change that fixes an issue)
- [X] New feature (change that adds new functionality)
- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)

## PR Checklist
- [X] I have followed the code style guidelines
- [X] I have performed a self-review of changes
- [X] I have compiled all hdl projects and libraries affected by this PR
- [ ] I have tested in hardware affected projects, at least on relevant boards
- [ ] I have commented my code, at least hard-to-understand parts
- [X] I have signed off all commits from this PR
- [X] I have updated the documentation (wiki pages, ReadMe files, Copyright etc)
- [ ] I have not introduced new Warnings/Critical Warnings on compilation
- [ ] I have added new hdl testbenches or updated existing ones

IuliaCMoldovan commented:
In this HDL documentation we won't upload pictures of the setup (that will be in the future in the [system level documentation](https://analogdevicesinc.github.io/documentation/eval/user-guide/index.html)), so if you could please make sure that the jumper and switches settings are written in the documentation, and the pictures to be removed.

actorreno commented:
v2:

- removed svg (pictures of setup) files not needed anymore

dc277a/index.rst:

- removed code showing system setup, made sure jumpers and switches are written

actorreno commented:
v3

dc2677a/index.rst:
- changed cyclone 5 reference and based it on architecture.rst
- removed empty line
- added missing ref utilcpack2
- removed a symbol on LTC2358-18 (trademark symobol?)
- added :adi:'' to some lines
- updated list from * to -
- adjusted length of ===== to match longest table entry
- removed the line "linux support:"
- removed the starting '/' in a directory

actorreno commented on docs/projects/dc2677a/index.rst:
```diff
@@ -0,0 +1,215 @@
+
+Software related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Linux support:
```
do we only add this line if there are both linux and no-os support to separate the entries?
or just total removal in general?

IuliaCMoldovan replied:
The first one; since there's only Linux, no need to specify it

actorreno commented on docs/projects/dc2677a/index.rst:
```diff
@@ -0,0 +1,215 @@
+surges, the :adi:`DC2677A` is capable of surviving extreme abuse.
+
+This reference design includes the LTC2358-18 SAR ADC, ADA4522-2 dual zero-drift
+operational amplifier, and an LT6658 dual-output, high current reference for
```
I thought the :adi:`<insert device here>` is only for the device in question, here it is dc2677a.
if the other ADI parts lt6658 and ada4522-2 is suggested to be inside the "adi::'<>', am curious why LTC2358-18 is not included

IuliaCMoldovan replied:
It's not a rule, but at least one time to have a link to a part. I didn't say for the LTC2358 because it is already in the Supported devices list.
And about the Registered mark sign, we've had some issues with these types of signs when generating the PDF version of documentation
